components.json

```
@@ -913,6 +913,11 @@
 			"alias": "pbfasm",
 			"owner": "HeX0R101"
 		},
+		"purescript": {
+			"title": "PureScript",
+			"alias": "purs",
+			"owner": "Sri Harsha Chilakapati"
```

We use this to link to your GitHub profile, so please use your GitHub username.

```suggestion
			"owner": "sriharshachilakapati"
```